### ISSUE TITLE
IOS客户端在中国不可用

### ISSUE THREAD

**Issue by crendy** [tinode/chat#852]
IOS客户端在中国不可用

**Comment by or-else** [comment]
Yes. China does not allow apps with `CallKit`. We had to remove the app from Chinese market.

**Comment by or-else** [comment]
https://developer.apple.com/forums/thread/103083

**Comment by crendy** [comment]
> https://developer.apple.com/forums/thread/103083

那现在有相应的解决方案吗？

**Comment by or-else** [comment]
The solution would be to disable the CallKit when the app is used in China. That's work. We don't have the resources to do it. It has to wait until someone either sends a PR or pays for it.
